front_end/html/edit_problem.html

```diff
@@ -240,8 +240,7 @@ <h2>Create Problem</h2>
         <a href="/delete_problem/{{ course_basics["id"] }}/{{ assignment_basics["id"] }}/{{ problem_basics["id"] }}">Delete problem</a> |
         <a href="/delete_problem_submissions/{{ course_basics["id"] }}/{{ assignment_basics["id"] }}/{{ problem_basics["id"] }}">Delete problem submissions</a>
       {% end %}
-    </p>
-
+    </p></br>
```

Comment:
There's actually no such thing as a closing br tag. But you can have a self-closing tag`<br />`. Perhaps that's what you meant. Please play with it a bit more to see what will work. Please also check on two different browsers to make sure it looks right on both.